climada/hazard/trop_cyclone.py

```diff
@@ -37,7 +37,7 @@
 from climada.hazard.tc_clim_change import get_knutson_criterion, calc_scale_knutson
 from climada.hazard.centroids.centr import Centroids
 from climada.util import ureg
-from climada.util.coordinates import dist_approx, lon_bounds, lon_normalize
+import climada.util.coordinates as co
```

Comment:
I used this PR to unify the shorthand for climada.util.coordinates everywhere in the whole project. The shorthand co is taken from the base modules in the hazard and exposures subpackages.